README.md

```diff
@@ -51,7 +52,7 @@ libp2p-switch is used by [libp2p](https://github.com/libp2p/js-libp2p) but it ca
 ```JavaScript
 const switch = require('libp2p-switch')
 
-const sw = new switch(peerInfo [, peerBook])
+const sw = new switch(peerInfo [, peerBook [, options]])
```

Comment:
This makes it look like options only exists in the context of passing peerBook. What about changing this to just a single options object and assert on its properties?

Reply:
@diasdavid can do this, but will be a breaking API change..

Reply:
Wait, I see that a peerBook is always required. The only optional argument should be `options`.